### PR TITLE
feat: Extend the descriptions of manifests

### PR DESCRIPTION
**Pull request by a-thaler**
## Description

Changes proposed in this pull request:

- added missing CRDa ttribute descriptions
- fixed examples for CRDs

## Related Issues and Documents

Closes:

Related issues:

## Traceability

- [ ] The PR is linked to a GitHub Issue.
- [ ] New features have a milestone label set.
- [ ] New features have defined acceptance criteria in a corresponding GitHub Issue, and all criteria are satisfied with this PR.
- [ ] The corresponding GitHub Issue has a respective `area` label.
- [ ] The follow-up issues (if any) are linked in the `Related Issues` section.

## Testability

The feature is unit-tested:

- [ ] Yes.
- [ ] No, because unit tests are not needed.
- [ ] No, because of ...

The feature is e2e-tested:

- [ ] Yes.
- [ ] No, because e2e-tests are not needed.
- [ ] No, because of ...

Tests conducted for the PR:

## Codebase

- [ ] My code follows the [Effective Go](https://go.dev/doc/effective_go) style guidelines.
- [ ] The code was planned and designed following the defined architecture and the separation of concerns.
- [ ] The code has sufficient comments, particularly for all hard-to-understand areas.
- [ ] This PR adds value and shows no feature creep.
- [ ] I have augmented the test suite that proves my fix is effective or that my feature works.
- [ ] Adjusted the documentation if the change is user-facing.

### CODE REVIEW

File: apis/telemetry/v1alpha1/shared_types.go [hunk 1]
```diff
@@ -19,7 +20,7 @@ func (v *ValueType) IsDefined() bool {
 }
 
 type ValueFromSource struct {
-	// Refers to a key in a Secret. You must provide `name` and `namespace` of the Secret, as well as the name of the `key`.
+	// Refers to a value located in a Secret by a key. You must provide `name` and `namespace` of the Secret, as well as the name of the `key`.
```
Comment (NHingerl) [inline apis/telemetry/v1alpha1/shared_types.go:23]:
```suggestion
	// Refers to the value of a specific key in a Secret. You must provide `name` and `namespace` of the Secret, as well as the name of the `key`.
```

File: apis/telemetry/v1alpha1/shared_types.go [hunk 2]
```diff
@@ -28,9 +29,12 @@ func (v *ValueFromSource) IsSecretKeyRef() bool {
 }
 
 type SecretKeyRef struct {
-	Name      string `json:"name,omitempty"`
+	// The name of the Secret containing the referenced value
+	Name string `json:"name,omitempty"`
+	// The name of the namespace containing the Secret with the referenced value.
```
Comment (NHingerl) [inline apis/telemetry/v1alpha1/shared_types.go:34]:
```suggestion
	// The name of the Namespace containing the Secret with the referenced value.
```
Namespace should be [capitalized](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#camelcase).

**Review by NHingerl — APPROVED** [review]

**Review by skhalash — APPROVED** [review]

**Pull request merged**